package-lock.json

```diff
@@ -715,6 +716,90 @@
         }
       }
     },
+    "node_modules/@puppeteer/browsers/node_modules/ansi-styles": {
```

Comment:
Is it weird that all this `puppeteer` stuff is being added? 🤔

Reply:
It's a dependency of `@web/test-runner-chrome` which is a dependency of `@web/test-runner`... so not sure why it was ever _not_ here?

Reply:
Huh yeah, that's super weird. We'll see what the `update` job does 😓

Reply:
This was the result of me doing a fresh `npm install` with no lock file, so hopefully it does nothing!